oap-server/analyzer/log-analyzer/src/main/java/org/apache/skywalking/oap/log/analyzer/provider/log/listener/RecordSinkListener.java

```diff
@@ -53,6 +55,7 @@
  */
 @RequiredArgsConstructor
 public class RecordSinkListener implements LogSinkListener {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RecordSinkListener.class);
```

Comment:
Prefer `@Slf4j` annotation

Reply:
Class field `private final Log log = new Log()` conflicts with the `@Slf4j` generated field name `log`, should I change all `log` to another name?

Reply:
> Class field `private final Log log = new Log()` conflicts with the `@Slf4j` generated field name `log`, should I change all `log` to another name?

OK forgot this, let's keep this now